Pod/Classes/Lifecycle/Behaviors/Nav-Bar-Hairline-Fade/NavBarHairlineFadeBehavior.swift

```diff
@@ -45,13 +45,22 @@ public extension NavBarHairlineFadeBehavior {
         set { navBarHairlineFadeUpdater.hairline.thickness = newValue }
         get { return navBarHairlineFadeUpdater.hairline.thickness }
     }
+    
+    var logsContentOffset: Bool {
+        set { navBarHairlineFadeUpdater.logsContentOffset = newValue }
+        get { return navBarHairlineFadeUpdater.logsContentOffset }
+    }
 
 }
 
 fileprivate final class NavBarHairlineFadeUpdater: NSObject {
 
     var contentOffsetFadeRange: ClosedRange<CGFloat> = 0...100
 
+    // Toggle for debug purposes:
+    // Logs the scrollview's y-offset while scrolling which helps implementer tweak the fade range more easily.
+    var logsContentOffset: Bool = false
```

Comment:
#nth use `///` comments to make them show up in Xcode's doc viewer. Also add or move doc comment to the public property so it's more likely to be seen by users.